app/services/journal_row_converter.rb

```diff
@@ -0,0 +1,24 @@
+class JournalRowConverter
+
+  # Returns an array of generated journal_rows given a single order_detail.
+  def self.from_order_detail(order_detail, journal = nil)
+    [{
+      account: order_detail.product.account,
```

Comment:
In the case of journaling "account" has a different meaning that we use "Account".

Take a look at the NU journal extension. `order_detail.product.account` will be something like `"75430"`, which is an important part of what goes into NU financials. There are some other fields that get populated in NU via `order_detail.account`:

```
activity: order_detail.account.activity,
chart_field1: order_detail.account.chart_field1,
dept: order_detail.account.dept,
...
```